wiki/Article_Styling_Criteria/en.md

```diff
@@ -472,7 +472,7 @@ Use `e.g.` when abbreviating "for example" and use `i.e.` for "that is". Side no
 
 ---
 
-When pluralizing acronyms and/or abbreviations, use an apostrophe followed by the letter `s` (`'s`) (e.g. `BN's`, `QAT's`, `GMT's`). If this writing style appears foreign or weird to you, try rewriting the sentence to avoid doing so in the first place.
+When pluralizing acronyms and/or abbreviations, do not use an apostrophe followed by the letter `s` (`'s`) (e.g. `BNs`, `QATs`, `GMTs`).

```

Comment:
Change to:
```
Do not pluralize acronyms and/or abbreviations.
```

Reply:
Probably the best approach, yeah. 'the BN, the GMT, the QAT' works better

Reply:
missed this, fixed